packages/react-server/core/__tests__/integration/redirectForward/RedirectForwardSpec.js

```diff
@@ -1,12 +1,12 @@
-var helper = require("../specRuntime/testHelper"),
+var helper = require("../../../test/specRuntime/testHelper"),
```

Comment:
Seems like that would work, assuming `lerna` does sane things with local dependencies. I'll give it a try.